packages/flutter/lib/src/material/text_field.dart

```diff
@@ -786,6 +788,12 @@ class TextField extends StatefulWidget {
   /// configuration, then [materialMisspelledTextStyle] is used by default.
   final SpellCheckConfiguration? spellCheckConfiguration;
 
+  /// {@macro flutter.rendering.editable.RenderEditablePainter}
```

Comment:
according to dart doc there's no such macro.

Reply:
If you want to draw cursors, would it be possible to just use a stack and position the cursor and the text field on the stack, instead of using these painters?

Reply:
Fixed doc macro.

As for using stack, that's challenge because:
1) Positioning cursor needs to call `RenderEdtiable.getLocalRectForCaret`. It is impossible to get the RenderEditable without using some GlabalKey.
2) It is hard to repaint the cursor when text layout changes (resizing window). Maybe CompositedTransformFollower can work, but seems cumbersome.